components/automate-ui/src/app/pages/project/list/project-list.component.ts

```diff
@@ -82,11 +90,26 @@ export class ProjectListComponent implements OnInit, OnDestroy {
       }))
       .subscribe(() => {
         this.creatingProject = false;
+
+        const createTeamsWasChecked = this.createProjectForm.controls['addTeams'].value;
```

Comment:
I don't know if and where we've settled on wrt. `this.createProjectForm.controls['addTeams'].value` vs `this.createProjectForm.controls.addTeams.value`

Reply:
Yeah no idea which is considered more idiomatic. Would be a good #ui-team question.